src/js/datepicker.js

```diff
@@ -110,9 +110,13 @@
 
         this.opts = $.extend(true, {}, defaults, options, this.$el.data());
 
-        if ($body == undefined) {
-            $body = $('body');
-        }
+        if (this.opts.renderTo) {
+            $body = $(this.opts.renderTo);
+        } else {
+            if ($body == undefined) {
```

Comment:
This is still hasn't been addressed

Reply:
@dennisbot The link 404's now, but there is no code to handle when `If the selector fails to match` as requested above. Sorry I missed your reply

Reply:
I think Javier was referring to this:

```javascript
$domContainer = $(this.opts.renderTo);
```

Which is unsafe because if `this.opts.renderTo` didn't have a valid element as a container, it would crash. Can we update it to something like this?

```javascript
if (this.opts.renderTo) {
  throw "Invalid container element provided";
}
$domContainer = $(this.opts.renderTo);
```